extension/source/Confirm/TransactionCard.tsx

```diff
@@ -37,7 +37,16 @@ const TransactionCard: React.FC<SendTransactionParams> = ({
         <div className="break-all">
           details:{' '}
           <span className="font-bold">{loading ? 'loading...' : method}</span>
-          <div className="text-[9pt] mt-2 font-normal">{data}</div>
+          <div className="text-[9pt] mt-2 font-mono">
+            {loading
+              ? 'loading params...'
+              : // prettier-ignore
```

Comment:
was getting too strict with number of spaces to leave at each line hence ignored